Gaijinizer patch/Map223.txt

```diff
@@ -106,21 +119,25 @@ There is no milking vagina, Her will not grow her hair.#MTLed
 
 だから、未だに笑った顔を見たことがない……。
 > CONTEXT: Map223/events/7/pages/0/73/Dialogue
-This body seems to have expression muscles though
-The influence of the previous body, the expression does not
-change much. So I have never seen a laughing face ....#MTLed
+This new body supposedly has muscles for facial
+expressions, but maybe because she's still influenced by
+her previous body, her expression never changes. I've
+never once seen her smile.#RonIffyWording
 > END STRING
```

Comment:
Some rewording:
_**This new body supposedly has muscles for facial
expressions... but maybe because she's so used to
her previous body, her face doesn't move. I've
never once seen her smile.**_